board/app/controllers/user_controller.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+class UserController extends AppController{
+
+    public function register() 
```

Comment:
Please fix indentions of the codes inside this function :bow: 
Some are a bit confusing since they are not indented properly, that at first glance I thought the function ended on Line 9 :sweat_smile: